tembo-cli/src/cmd/apply.rs

```diff
@@ -944,7 +944,7 @@ fn get_extensions(
                 vec![ExtensionInstallLocation {
                     database: Some("postgres".to_string()),
                     schema: None,
-                    version: version,
+                    version: Some(version),
```

Comment:
`version` is already an `Option<String>`, so you can do this:

```suggestion
                    version,
```